### PR TITLE
eval: add recent eval result

### PR DESCRIPTION
**Pull request by asamal4**
## Description

Add recent eval result. [OLS-1079](https://issues.redhat.com//browse/OLS-1079)

### CODE REVIEW

File: scripts/evaluation/eval_data/result/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Here we ask a judge model to score/grade response generated by OLS against the pre-defined/ground-truth answer.
+![Similarity Score by LLM](model_evaluation_result-answer_similarity_llm.png)
+#### Cosine Similarity
+Cosine similarity between vector representation (generated by embedding model) of ground-truth answer & OLS response.
+![Cosine Similarity](model_evaluation_result-cos_score.png)
+#### Answer Relevancy
+Here we generate multiple questions from the response using a Model and calculate average cosine similarity score between original query & generated question.
+![Answer Relevancy score](model_evaluation_result-answer_relevancy.png)
+#### Rouge-L
+This score purely based on stem words. Here longest common subsequence (LCS) is being considered (not necessarily in order) between predefined answer and OLS response.
```
Comment (JoaoFula) [inline scripts/evaluation/eval_data/result/README.md:29]: This **scores** purely based on stem words

Comment (JoaoFula) [reply scripts/evaluation/eval_data/result/README.md:29]: Here **the** longest common subsequence...

Comment (JoaoFula) [reply scripts/evaluation/eval_data/result/README.md:29]: between **the** predefined answer and **the** OLS response

**Comment by tisnik** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

**Pull request merged**